### PR TITLE
Add multimodal agent with fastRAG and Haystack

### PR DESCRIPTION
**Pull request by bilgeyucel**

### CODE REVIEW

File: notebooks/multimodal_agent_with_fastrag_haystack.ipynb [hunk 1]
```diff
@@ -0,0 +1,2703 @@
+{
```
Comment (anakin87) [inline notebooks/multimodal_agent_with_fastrag_haystack.ipynb:1]: <p>We can say something like:</p><p>As an example, you can use the nutrition_data.json file provided in the fastRAG repository.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (anakin87) [inline notebooks/multimodal_agent_with_fastrag_haystack.ipynb:1]: <p>we should probably clarify that we are embedding the content, not the image. Right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (anakin87) [inline notebooks/multimodal_agent_with_fastrag_haystack.ipynb:1]: <p>can we show the original image to confirm the correctness of the response?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (anakin87) [inline notebooks/multimodal_agent_with_fastrag_haystack.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    agent_response = multimodal_agent.run("How many calories does cheerios have?")
</pre><p> </p><p>How many calories do cheerios have?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (bilgeyucel) [reply notebooks/multimodal_agent_with_fastrag_haystack.ipynb:1]: <p>hmmm good point</p>

**Pull request merged**